docs/faq.md

```diff
@@ -0,0 +1,38 @@
+## How can I cite Larq?
+
+If Larq helps you in your work or research, it would be great if you can cite it as follows:
+
+```
+@misc{larq,
+  author       = {Geiger, Lukas and Widdicombe, James and Bakhtiari, Arash and Helwegen, Koen and Heuss, Maria and Nusselder, Roeland},
+  title        = {Larq: An Open-Source Deep Learning Library for Training Binarized Neural Networks},
+  howpublished = {Web page},
+  url          = {https://larq.dev},
+  year         = {2019}
+}
+```
+
+If your paper is publicly available, feel free to also add it to the list of [Papers using Larq](/papers).
+
+## Can I add my own algorithm or model to Larq?
+
+Absolutely! If you have developed a new BNN or training method that you would like to share with the community, create a PR or get in touch with us. Make sure you check out the contribution guide. For entire models with pretrained weights [`larq-zoo`](https://github.com/plumerai/larq-zoo) is the correct place, everything else can be added to [`larq`](https://github.com/plumerai/larq) directly.
```

Comment:
Here I would also mention that larq is not only meant for BNNs but also for other quantized networks and that code also for those networks as well as models are welcome as well.